scrapy/spiders/__init__.py

```diff
@@ -89,6 +91,27 @@ def close(spider, reason):
         if callable(closed):
             return closed(reason)
 
+    @staticmethod
+    def idle(spider):
+        idled = getattr(spider, 'idled', None)
+        if callable(idled):
+            cb_response = idled()
```

Comment:
Nice :)
Simple unawareness. It happens that I forget about some utility function or other here after some time away from the codebase (sorry).